debezium-engine-app/src/test/java/org/yb/AppTest.java

```diff
@@ -0,0 +1,20 @@
+package org.yb;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
```

Comment:
Perhaps add a comment that this is just a dummy test, we can take up the implementation of some actual tests later on.